src/elements/content-picker/Footer.js

```diff
@@ -38,7 +38,7 @@ const Footer = ({
 }: Props) => (
     <footer className="bcp-footer">
         <div className="bcp-footer-left">
-            <Button className="bcp-selected" onClick={onSelectedClick} type="button">
+            <Button className="bcp-selected" onClick={onSelectedClick} type="button" data-testid="bcp-selected">
```

Comment:
Can we select this by its content, instead? Using `data-testid` is kind of a fallback solution.